### PR TITLE
fix(API): There was a letter outside the code formatting

### PR DESCRIPTION
**Pull request by paperclypse**

### CODE REVIEW

File: src/content/docs/apis/rest-api-v2/application-examples-v2/get-average-throughput-app-v2.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ curl -X GET "https://api.newrelic.com/v2/applications/${APPID}/metrics/data.json
 
 ## Web app throughput (single host) [#api-call-web-single]
 
-To find the average throughput from a single host, for a given time period, use a single command to get the metric `HttpDispatche`r`:requests_per_minute`, and specify the numeric $&#x7B;HOST}. To identify the host, use the [UI or the REST API](/docs/apis/rest-api-v2/requirements/listing-host-instance-application-server-ids#locating_host_id).
+To find the average throughput from a single host, for a given time period, use a single command to get the metric `HttpDispatcher`:requests_per_minute`, and specify the numeric $&#x7B;HOST}. To identify the host, use the [UI or the REST API](/docs/apis/rest-api-v2/requirements/listing-host-instance-application-server-ids#locating_host_id).
```
Comment (paperclypse) [inline src/content/docs/apis/rest-api-v2/application-examples-v2/get-average-throughput-app-v2.mdx:40]:
```suggestion
To find the average throughput from a single host, for a given time period, use a single command to get the metric `HttpDispatcher`:`requests_per_minute`, and specify the numeric $&#x7B;HOST}. To identify the host, use the [UI or the REST API](/docs/apis/rest-api-v2/requirements/listing-host-instance-application-server-ids#locating_host_id).
```

**Review by jeff-colucci — DISMISSED** [review]
Good catch!

**Comment by paperclypse** [comment]
NOW I'm done :)

**Review by jeff-colucci — APPROVED** [review]
Looks good part 2 :)

**Pull request merged**